src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/UpdateConsent/HealthCertifiedPersonUpdateConsentViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum HealthCertifiedPersonUpdateError: Error {
```

Comment:
I think this is redundant with the error I created in the resource. I think you can remove it and replace in the result handler by just Error.
Then I can replace it in my PR with the concrete error

Reply:
okay